site/docs/master/basic-install.md

```diff
@@ -0,0 +1,59 @@
+# Basic Install
+
+- [Basic Install](#basic-install)
+  - [Prerequisites](#prerequisites)
+  - [Install the CLI](#install-the-cli)
+    - [Option 1: macOS - Homebrew](#option-1-macos---homebrew)
+    - [Option 2: GitHub release](#option-2-github-release)
+  - [Install and configure the server components](#install-and-configure-the-server-components)
+
+Use this doc to get a basic installation of Velero.
+Refer [this document](customize-installation.md) to customize your installation.
+
+## Prerequisites
+
+- Access to a Kubernetes cluster, v1.10 or later, with DNS and container networking enabled.
+- `kubectl` installed locally
+
+Velero uses object storage to store backups and associated artifacts. It also optionally integrates with supported block storage systems to snapshot your persistent volumes. Before beginning the installation process, you should identify the object storage provider and optional block storage provider(s) you'll be using from the list of [compatible providers][0].
+
+There are supported storage providers for both cloud-provider environments and on-premises environments. For more details on on-premises scenarios, see the [on-premises documentation][2].
+
+## Install the CLI
+
+### Option 1: macOS - Homebrew
+
+On macOS, you can use [Homebrew](https://brew.sh) to install the `velero` client:
+
+```bash
+brew install velero
+```
+
+### Option 2: GitHub release
+
+1. Download the [latest release][1]'s tarball for your client platform.
+1. Extract the tarball:
+
+   ```bash
+   tar -xvf <RELEASE-TARBALL-NAME>.tar.gz
+   ```
+
+1. Move the extracted `velero` binary to somewhere in your `$PATH` (e.g. `/usr/local/bin` for most users).
+
+## Install and configure the server components
+
+There are two supported methods for installing the Velero server components:
+
+- the `velero install` CLI command
+- the [Helm chart](https://github.com/helm/charts/tree/master/stable/velero)
+
+Velero uses storage provider plugins to integrate with a variety of storage systems to support backup and snapshot operations. This requires at least one plugin to be added to the server installation. This can be done using the `--plugin` flag. Please refer to our [Plugins](3) documentation for more information.
```

Comment:
I'm trying to think about how to make this section as clear as possible. IMHO the main point to convey is that the server installation steps vary based on your storage provider, so you'll find the details in the documentation for your storage provider's plugin, links to which can be found on `supported-providers.md`.

I'd combine L50 and L52, something along the lines of:

"Velero uses storage provider plugins to integrate with a variety of storage systems to support backup and snapshot operations. The steps to install and configure the Velero server components along with the appropriate plugins are specific to your chosen storage provider.  To find installation instructions for your chosen storage provider, follow the documentation link for your provider at our [supported storage providers][insert link # here] page."

Wordsmithing welcome, but I think something along those lines focuses on the important point to convey here.  I'd leave details about the `--plugins` flag for the provider-specific instructions where it'll be documented in detail.

Reply:
I am cool w/ the way you've worded this :) I'll use that.